src/app/components/footer/footer.component.html

```diff
@@ -1,36 +1,47 @@
 <footer class="pb-0 pt-2 bg-light">
   <div class="container">
-    <div class="row justify-content-between">
-      <div class="col-md-4">
-          <span placement="top" [ngbTooltip]="tipContent" i18n tabindex="0">Program versions</span>
-      </div>
-      <div class="col-md-5 pull-right">
-        <div class="row">
-          <div class="col-md-6">
+    <div class="col-md-12">
```

Comment:
> Why is this a (bootstrap) column (of the size of the container)?

To make sure it takes up the entire width and thereby ends up on top of each other

Reply:
Okay, if it is equivalent maybe it would be easier to understand if it was a class "row". As I understand it, it would be the same visually, if not I am fine with it how it is.

Reply:
> Okay, if it is equivalent maybe it would be easier to understand if it was a class "row". As I understand it, it would be the same visually, if not I am fine with it how it is.

Yeah it would be visually the same, but for clarity wrapping them in `row` would be more consistent with how the rest of the code is structured so I think we can go head and wrap them.

Reply:
> > Why is this a (bootstrap) column (of the size of the container)?
> 
> To make sure it takes up the entire width and thereby ends up on separate "rows". We could of course also wrap them in actual row-containers. But it's not a necessity. @matsduf We should change the classes to `col-12`. The `md` part is redundant.

Updated.